content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -277,7 +277,8 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
 ## Arbitrary Controllers and Selectors
```

Comment:
```suggestion
## Arbitrary object kinds and arbitrary selectors {#arbitrary-controllers-and-selectors}
```

Reply:
As mentioned in https://github.com/kubernetes/website/pull/39196#discussion_r1100583119 I would prefer we use something like Workload resources. But in the title I would rather say `workloads` since it applies transitively to pods as well. 

Object kinds would also suggest that the PDB is managing other kinds and not just pods.

Also, do we want to duplicate the arbitrary word? Does it help with comprehensiveness/ambiguity?